docs/UserGuide.md

```diff
@@ -390,21 +388,10 @@ Possible Usage:
 
 #### Giving an assignment to all students in a module: `giveall`
 
-Adds an assignment to all students in the specified module .
+Gives an assignment with a deadline to all students in the specified module .
```

Comment:
I think you might have forgotten to remove 'with a deadline'

Reply:
yes thats correct, thanks for pointing it out